cmd/lint/main.go

```diff
@@ -121,6 +122,8 @@ var flagRules = []linter.FlagRule{
 			"destination-api-key",
 			"destination-api-secret",
 			"destination-bootstrap-server",
+			"gcp-import-custom-routes",
```

Comment:
Any way we can shorten this name?

Reply:
checked with @kabraham13 and shortened it to `gcp-routes`